.ci/.jenkins_tav.yml

```diff
@@ -1,29 +1,30 @@
 TAV:
+  - @elastic/elasticsearch
```

Comment:
Sorry for the diff... I only added this line, but sorted all entries alphabetically